eeg_cofitage/README

```diff
@@ -0,0 +1,71 @@
+﻿Overview
+--------
+
+This is the “COFITAGE” dataset. The aim is to collect sleep EEG data from 100 to 120 healthy older 
+participants (aged 50-70), in order to quantify sleep disruption, evaluate its impact on cognition
+and assess its relationship with brain amyloid and tau proteins deposition in preclinical AD.
+
+We would like to determine which sleep-wake regulation parameters are associated with biomarkers 
+and cognitive status at baseline and at a 24 months follow-up of cognitive performances.
+
+
+Citing this dataset
+-------------------
+
+This dataset and related analyses are unpublished. Please do not use/cite given data other than
+BIDS dataset example.
```

Comment:
Do you know when the dataset and analyses will be published? Perhaps you can add an estimate here.

Reply:
Unfortunately I can't make an estimation. The tool I use not yet officially adapted so the full dataset is not processed yet, also the declaration that our subjects signing are incompatible with BIDS list of license. So this part also must be resolved.

Reply:
Could you elaborate this part a bit? What LICENSE did your subjects agree to regarding data sharing? and which list of BIDS compatible licenses are you referring to?

What are the steps you are planning for resolution of this issue?

Reply:
1. Validate tool
 - Conversion algorithm already been validated by exterior laboratory
 - Bidsifier, I will consider it validated once my example will be merged
2. Officially accept and integrate into workflow
 - It has been tested in test workflow, so full deployment should take no more than a week
3. Process all existing data
4. Decide if data should be public, where to publish it, and under which licence

But most importantly, the data taking is not yet finished.